test/blackbox-tests/test-cases/vlib/run.t

```diff
@@ -354,20 +355,21 @@ Include variants and implementation information in dune-package
    (main_module_name Vlib)
    (modes byte native)
    (modules
-    (alias_module
-     (name Vlib)
-     (obj_name vlib)
-     (visibility public)
-     (kind alias)
-     (impl))
-    (main_module_name Vlib)
-    (modules
-     ((name Vmod)
-      (obj_name vlib__Vmod)
+    (wrapped
+     (main_module_name Vlib)
+     (modules
+      ((name Vmod)
+       (obj_name vlib__Vmod)
+       (visibility public)
+       (kind virtual)
+       (intf)))
+     (alias_module
+      (name Vlib)
+      (obj_name vlib)
       (visibility public)
-      (kind virtual)
-      (intf)))
-    (wrapped true)))
+      (kind alias)
+      (impl))
+     (mode true))))
```

Comment:
Actually, this looks odd. What does `(mode true)` mean?

Reply:
I think we should keep the name `Wrapped` for the module and field name. Alternatively, we could rename the constructors as follow:

- `Simple true -> Wrapped`
- `Simple false -> Unwrapped`
- `Yes_with_transition x -> Wrapped_with_transition x`

Reply:
Okay.

Reply:
Actually, let's keep the renaming to another PR. It touches too much unrelated code.

I renamed the field to `wrapped` however.